string uuid
bool running
string status

string STATUS_RUNNING = "running"
string STATUS_IDLE = "idle"
string STATUS_PAUSED = "pause"
string STATUS_ERROR = "error"